pylib/zeus/zeus.py

```diff
@@ -52,6 +93,26 @@ def _Init(self):
       parser.set_defaults(func=self._GetHandler(cmd, 'run'))
 
     Flags.InitArgs()
+    if not all((Flags.ARGS.id, Flags.ARGS.root)):
+      if Flags.ARGS.task != ['...']:
+          tasks_dirs = map(
+            lambda t: os.path.abspath(t) if os.path.isdir(t)
+                      else os.path.dirname(os.path.abspath(t)),
+            Flags.ARGS.task
+          )
+          base_dir = os.path.commonpath(tasks_dirs)
+          try:
+            zeus_executable = find_zeus_executable(base_dir)
+          except FileNotFoundError:
+            zeus_executable = None
+      else:
+          zeus_executable = find_zeus_executable(os.getcwd())
```

Comment:
Aaaaah - ok - it took me a while to figure this out. So we've found the correct thing to run, and we're replacing the current process with that one. Got it.
"the current process" confused me. it's more like "this process" ; ok. ok.

Reply:
it's chatgpt :)

Reply:
I didn't look like a human. I assumed it was you. ;)